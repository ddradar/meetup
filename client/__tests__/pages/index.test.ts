import { createLocalVue, mount } from '@vue/test-utils'
import Buefy from 'buefy'

import IndexPage from '~/pages/index.vue'

const localVue = createLocalVue()
localVue.use(Buefy)

describe('/pages/index.vue', () => {
  describe('snapshot test', () => {
    test('renders correctly', () => {
      // Arrange - Act
      const wrapper = mount(IndexPage, { localVue, stubs: ['card'] })

      // Assert
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
