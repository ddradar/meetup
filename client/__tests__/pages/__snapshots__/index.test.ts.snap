// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/pages/index.vue snapshot test renders correctly 1`] = `
<section
  class="section"
>
  <div
    class="columns is-mobile"
  >
    <card-stub
      icon="github"
      title="Free"
    >
      
      Open source on 
      <a
        href="https://github.com/buefy/buefy"
      >
         GitHub 
      </a>
    </card-stub>
     
    <card-stub
      icon="cellphone-link"
      title="Responsive"
    >
      <b
        class="has-text-grey"
      >
         Every 
      </b>
       component is responsive
    
    </card-stub>
     
    <card-stub
      icon="alert-decagram"
      title="Modern"
    >
      
      Built with 
      <a
        href="https://vuejs.org/"
      >
         Vue.js 
      </a>
       and
      
      <a
        href="http://bulma.io/"
      >
         Bulma 
      </a>
    </card-stub>
     
    <card-stub
      icon="arrange-bring-to-front"
      title="Lightweight"
    >
      
      No other internal dependency
    
    </card-stub>
  </div>
</section>
`;
